.npmignore

```diff
@@ -1,2 +1,22 @@
 utils
-.git
\ No newline at end of file
+.git
+
+# ignore the .ts files
+*.ts
+*.js.map
```

Comment:
@czirker So, these are the source maps. You don't want those included?

Reply:
I went back and forth on this.
I was think because I'm not exporting the source ts files the lines aren't helpful to the person debugging.
What do you think?

Reply:
We are excluding the .ts files because it was causing issues when another project tried to webpack the module.  It was trying to compile the leo-sdk ts files and throwing errors.  So I figured we can just not export them

Reply:
@jgrantr ^

Reply:
@czirker So, the .js.map file is the companion to the .js file, and if you include `inlineSources` in your tsconfig, you actually don't need the source TS files, it will put them inline and allow people using this to debug it.

Reply:
Cool.  I'll add them back